tasks/rabbitmq-provision.yml

```diff
@@ -23,12 +22,5 @@ run:
       gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
       gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT}
 
-      apt-get install -y procps  # NB: procps is used by Helm
-      curl -L https://git.io/get_helm.sh | bash -s -- --version v2.16.3
-      helm init --client-only
-      helm plugin install https://github.com/rimusz/helm-tiller
-
       cd census-rm-kubernetes-dependencies-repo
-      export HELM_TILLER_SILENT=true
-
-      ENV=${ENV} RABBITMQ_CONFIG_VALUES_FILE=${RABBITMQ_CONFIG_VALUES_FILE} ./prod-setup-dependencies.sh
+      ENV=${ENV} ./setup-rabbitmq.sh
```

Comment:
This appears to still be running the 'set up rabbit from scratch' script, rather than just updating the config/definitions

Reply:
The script does an update if the crd already exists or an install if it doesn't exist. So it should only update once its already been initially set up in the project.